### PR TITLE
Sprint Challenge Completed, Kev G

### PR DESCRIPTION
**Pull request by TheObserverofOne**

### CODE REVIEW

File: Answers.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+If you saw this HTML: <div class="box box1 box2 box3"></div> which class has the most specificity weight?  box3 as it appears last
```
Comment (lefrenk) [inline Answers.md]: 👍

File: Answers.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+If you saw this HTML: <div class="box box1 box2 box3"></div> which class has the most specificity weight?  box3 as it appears last                                                     
+Describe the difference between display: block; and display: inline;.  display:block is helpful in such cases as taking links out of their default inline property and setting them to use up the entire amount of space available, whereas display: inline is useful for taking elements out of their own line and putting them all on the same horizontal line with them only taking up minimal space.                                              
+While using flexbox, what axis are you using when you use the property: align-items: center? cross axis
```
Comment (lefrenk) [inline Answers.md]: 🙌

File: css/index.css [hunk 3]
```diff
@@ -96,26 +144,30 @@ p {
...
     display: flex;
     align-items: center;
     justify-content: center;
+    text-align: center;
 }
 
 .bottom-content {
     display: flex;
-    margin: 0 2% 20px;
+    margin: -4px 1% 25px 4.3%;
```
Comment (lefrenk) [inline css/index.css]: 🤔

Comment (TheObserverofOne) [reply css/index.css]: Pixel perfection, haha... or tried it at least.

File: css/index.css [hunk 4]
```diff
@@ -57,35 +60,80 @@ html, body {
...
 }
 
 p {
     line-height: 1.4;
+    font-size: 1.6rem;
 }
 
 .container {
-    width: 800px;
+    max-width: 800px;
```
Comment (lefrenk) [inline css/index.css]: 👍

File: index.html [hunk 5]
```diff
@@ -13,7 +15,24 @@
...
+                    <li><a href="#">Blog</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </nav>
+            
+        </header>
+
         <section class="top-content">
+
+            <img class="top-image" src="img/jumbo.jpg" alt="Blurred highway traffic image.">
```
Comment (lefrenk) [inline index.html]: Love the descriptive `alt` attribute! :)

File: css/index.css [hunk 6]
```diff
@@ -57,35 +60,80 @@ html, body {
...
+    margin: 5px 1.5% 0 2%;
+}
+
 .top-content {
     display: flex;
     flex-wrap: wrap;
     justify-content: space-evenly;
-    margin-bottom: 20px;
     border-bottom: 1px dashed black;
+    margin: 10px 0 20px 2.3%;
```
Comment (lefrenk) [inline css/index.css]: The `2.3%` margin-left here is giving your jumbotron image spacing on the left side in the 400px view. You could override that with your 400px media query.

File: css/index.css [hunk 7]
```diff
@@ -137,10 +191,156 @@ footer nav {
...
+        font-size: 1.8rem;
+        width: 100%;
+        padding: 3% 6% 2%;
+    }
+
+    .middle-content .boxes {
+        flex-direction: row-reverse;
+    }
+
+    .middle-content .boxes .box {
```
Comment (lefrenk) [inline css/index.css]: You can give the boxes a height of `100px` so that they keep their square shape!

File: css/index.css [hunk 8]
```diff
@@ -137,10 +191,156 @@ footer nav {
...
+
+    .middle-content .boxes .box9 {
+        background: crimson;
+    }
+
+}
+
+
+
+@media (max-width: 400px) {
```
Comment (lefrenk) [inline css/index.css]: Excellent work here! 👌

**Comment by lefrenk** [comment]
Wow Kev! Fantastic work! You've almost nailed it with each media query and you clearly have a good grasp for the material this week. Kudos to you! 👌

- Site looks great at all media queries ✔️
- Code is consistent with HTML/CSS idioms and style ✔️
- Good grasp of the sprint material ✔️

I'd like to see you commit more often with better descriptors.

Not much else to say. Keep up the good work. Please close your pull request when you get a chance. Thanks!

**Pull request closed without merging**